stacker/blueprints/testutil.py

```diff
@@ -0,0 +1,34 @@
+import difflib
+import unittest
+
+
+def diff(a, b):
+    """A human readable differ."""
+    return '\n'.join(
+        list(
+            difflib.Differ().compare(
+                a.splitlines(),
+                b.splitlines()
+            )
+        )
+    )
+
+
+class BlueprintTestCase(unittest.TestCase):
+    def assertEqualsDiff(self, a, b):  # noqa: N802
+        self.assertEquals(a, b, diff(a, b))
+
+    def assertRenderedBlueprint(self, blueprint, dump=True,  # noqa: N802
+                                expected=None):
+        expected_output = "tests/fixtures/blueprints/%s.json" % blueprint.name
```

Comment:
I feel like this path should probably be configurable, maybe with a class variable (is there a common "fixture path" for unittest?).